schemas/core/3.3.0/version.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title" :  "Version",
+  "description" :  "Version message",
+  "properties" : {
+    "mId" : { "$ref": "definitions.json#/message_id" },
+    "step" : {
+      "description" : "Request or Response step",
+      "type" : "string",
+      "enum" : ["Request", "Response"]
+    },
+    "RSMP" : {
+      "description" : "Supported RSMP versions",
+      "type" : "array",
+      "items" : {
+        "type" : "object",
+        "properties" : {
+          "vers" : {
+            "$ref": "definitions.json#/version",
+            "description" : "RSMP version"
+          }
+        },
+        "required" : ["vers"],
+        "additionalProperties" : false
```

Comment:
additional properties should be allowed in Version messages, as the core spec specifically states that unknown attributes in Version messages must be ignored